rotkehlchen/globaldb/updates.py

```diff
@@ -300,74 +305,219 @@ def _parse_full_insert(self, insert_text: str) -> AssetData:
             protocol=protocol,
         )
 
+    def _process_asset_collection(
```

Comment:
Question. In the current implementation identifiers of asset collections are just sequential integers 0, 1, 2, ...

We don't allow this yet, but if a user at some point adds a collection, and then later we run an asset collections update, then ... what identifiers will we use for the collections in the update?

Maybe we can later do something like a flag in the table that would help to distinguish, but not sure what is best to do here